### DISCUSSION TITLE
History tab improvement suggestions

### DISCUSSION THREAD

**Discussion by RoboMagus** [home-assistant/frontend#13220] [Other feature requests]
The new addition to the history tab that allows selection of multiple devices / entities / areas is amazing. Much love for the Dev that made that possible!

## Wildcard entity filter
However I'm missing the option to use wildcards in the selection query, much like what is possible in the entity filter of the developer tools. E.g. to query for `sensor.*linkquality` to show all my Zigbee device linkquality sensors in one go.

This becomes especially useful, as the width of the dropdown selection for entities is *severely* limited, making it a guessing game what the end of the entity name would be when there are numerous sensors that start their name with the same string.

## Mouse-over for (entity-)selection dropdown
On that note, it'd be nice to have a mouse-over view for the dropdown entries that would show us the full name, instead of only what is able to fit within the width constraints of the dropdown menu.

**Comment by wigster** [comment]
on this theme, it would be great if the drop down reopened with the last query instead of reset to show all entities, so that you could easily add the other entity in the filtered list if you picked the wrong one the first time.
